pixl_dcmd/tests/test_main.py

```diff
@@ -137,20 +139,87 @@ def test_can_nifti_convert_post_anonymisation(
     assert np.all(anon_nifti.get_fdata() == ident_nifti.get_fdata())
 
 
-def test_merge_tag_schemes_single_file():
-    tag_ops_file = (
-        pathlib.Path(__file__).parents[2]
-        / "projects/configs/tag-operations/test-extract-uclh-omop-cdm.yaml"
+BASE_ONLY_TAG_OPS = TagOperations(base="base.yaml", manufacturer_overrides=None)
+
+
+@pytest.fixture()
+def base_only_tag_scheme() -> TagOperations:
+    return TagOperations(base=BASE_TAGS_FILE, manufacturer_overrides=None)
+
+
+def test_merge_base_only_tags(base_only_tag_scheme):
+    """
+    GIVEN TagOperations with only a base file
+    WHEN the tag schemes are merged
+    THEN the result should be the same as the base file
+    """
+    tags = merge_tag_schemes(base_only_tag_scheme)
+    expected = _load_scheme(BASE_TAGS_FILE)
+    assert tags == expected
+
+
+@pytest.fixture(scope="module")
+def tag_ops_with_manufacturer_overrides(tmp_path_factory):
+    """
+    TagOperations with a base file and manufacturer overrides, where the base file has 3 tags
+    and the manufacturer overrides ovverrides 1 of them and has 2 extra tags.
+    This is intetnionally not added in conftest.py because the `PROJECT_CONFIG_DIR` envvar nees to
+    be set before importing core.project_config.TagOperations.
+    """
+    base_tags = [
+        {"name": "tag1", "group": 0x001, "element": 0x1000, "op": "delete"},
+        {"name": "tag2", "group": 0x002, "element": 0x1001, "op": "delete"},
+        {"name": "tag3", "group": 0x003, "element": 0x1002, "op": "delete"},
+    ]
+    manufacturer_overrides_tags = [
+        {
+            "manufacturer": "manufacturer_1",
+            "tags": [
+                # Override tag1 for manufacturer 1
+                {"name": "tag1", "group": 0x001, "element": 0x1000, "op": "keep"},
+                {"name": "tag4", "group": 0x004, "element": 0x1011, "op": "delete"},
+                {"name": "tag5", "group": 0x005, "element": 0x1012, "op": "delete"},
+            ],
+        },
+        {
+            "manufacturer": "manufacturer_2",
+            "tags": [
+                {"name": "tag6", "group": 0x006, "element": 0x1100, "op": "keep"},
+                {"name": "tag7", "group": 0x007, "element": 0x1111, "op": "delete"},
+                # Override tag3 for manufacturer 2
+                {"name": "tag3", "group": 0x003, "element": 0x1002, "op": "keep"},
+            ],
+        },
+    ]
+
+    tmpdir = tmp_path_factory.mktemp("tag-operations")
+    base_tags_path = tmpdir / "base.yaml"
+    with open(base_tags_path, "w") as f:
+        f.write(yaml.dump(base_tags))
+    manufacturer_overrides_path = tmpdir / "manufacturer_overrides.yaml"
+    with open(manufacturer_overrides_path, "w") as f:
+        f.write(yaml.dump(manufacturer_overrides_tags))
+
+    return TagOperations(
+        base=base_tags_path, manufacturer_overrides=manufacturer_overrides_path
```

Comment:
mentioned it before but seems strange that these are tests in main, where its testing a specific part of functionality rather than how users interact with dcmd. If we move out the `load_scheme` into a project config module that'd make more sense IMHO